README.md

```diff
@@ -121,6 +123,13 @@ service = Expeditor::Service.new(
 command = Expeditor::Command.new(service: service) do
   ...
 end
+
+# v0.4.0 or later
```

Comment:
Do you add version information to README whenever you release a new feature? In my opinion, master branch's README doesn't need to include the information for old version since it is achieved in CHANGELOG.